### DISCUSSION TITLE
after push and commit PR doesn't reflecting to any changes

### DISCUSSION THREAD

**Discussion by GeorgiSwiftDeveloper** [community/community#30462] [Pull Requests]
Hi,

I created pr and then did some code refactor after which I push my commit to the repo but for some reason it's not showing up on PR that there are new commits and it's still tracking old commit. 

Thank you

**Comment by NikhilMath** [comment]
can you please link me to the PR that didn't have some commits tracked in it?

**Comment by GeorgiSwiftDeveloper** [comment]
When I'm checking my PR it's showing 4 commits and not showing the latest one but when trying to create a new PR from SourceTree all 5 commits showing up and it's not allowing to create a new PR since it's already available .

<img width="961" alt="Screen Shot 2022-08-22 at 4 28 36 PM" src="https://user-images.githubusercontent.com/42850605/186037246-10df8425-1570-4611-9290-c697572f7b5f.png">
<img width="991" alt="Screen Shot 2022-08-22 at 4 29 08 PM" src="https://user-images.githubusercontent.com/42850605/186037248-71b499b9-94af-47e2-a99c-80f4d53f5a59.png">
